.travis.yml

```diff
@@ -61,6 +61,12 @@ jobs:
       node_js: '8'
       env: ELASTIC_APM_ASYNC_HOOKS=false
 
+    # Docs
+    -
+      script: npm run docs:build
+      name: "Build Docs"
+      perl: '5.26'
```

Comment:
@bmorelli25 I'm not sure if perl is still needed?

Reply:
Good question. This is now using the new script, so I would guess not. I'm not 100% sure though

Reply:
👍 I'll try to remove it and see what happens to the tests

Reply:
It worked 💯